shared/constants/settings.js

```diff
@@ -75,6 +75,12 @@ export const makeState: I.RecordFactory<Types._State> = I.Record({
   waitingForResponse: false,
 })
 
+export const getPushTokenForLogSend = (state: Object) => {
+  return I.Map({
```

Comment:
why make a map and convert it to an object?

Reply:
i was following the existing call below: https://github.com/keybase/client/pull/17591/files#diff-1d9ec48879ceb528287d545487be4738R89

Reply:
thats cause it had hierarchical immutable structures, which this does not. can be
```
=> ({pushToken: state.push.token})
```

Reply:
got it, thanks!